jest.config.js

```diff
@@ -5,5 +5,14 @@ module.exports = {
     __testing__: true
   },
   roots: ['app/javascript'],
-  testMatch: ['**/*.test.js']
+  setupFiles: ['./config/jest.setup.js'],
+  testRegex: '(/__tests__/.*|(\\.|/)(test))\\.(js?|ts?)$',
```

Comment:
Would it make sense to name the dir `specs/` as with old js? Or is this a new convention to use `__tests__/`?

Reply:
Also, we may want to support `*.spec.js` since that's what SUI does.

Reply:
You can either put it into new folder `__tests__` or leave them in any folder, that is jus optional. However you have to name your file `${whatever}.test.[js|ts]` this is because there might be clash between jasmine old tests and new one's written in jest.

Reply:
Or no, there is root directory, I'll add option to use spec or test as part of file.